plugins/arangodbendorsements/main_test.go

```diff
@@ -81,7 +81,8 @@ func TestInitStore(t *testing.T) {
 
 			err := fetcher.Init(argList)
 			if err != nil {
-				assert.EqualErrorf(t, err, test.wantErr, "received error got != want (%s, %s)", err.Error(), test.wantErr)
+				//assert.EqualErrorf(t, err, test.wantErr, "received error got != want (%s, %s)", err.Error(), test.wantErr)
```

Comment:
this line should go

Reply:
Yes it was a mistake!, I corrected it immediately!